project/Dist.scala

```diff
@@ -26,10 +27,7 @@ object Dist {
       val zincJar = (packageBin in Compile).value
       val zincArtifact = (artifact in Compile).value
       zincJar -> filename(zincArtifact)
-    },
-    artifact in packageZipTarball in Universal :=  Artifact("zinc", "tgz", "tgz"),
```

Comment:
Since the tarballs are not published to an Ivy/Maven repo and the code that used that setting has been removed, this setting can safely be removed.